### ISSUE TITLE
npc follows me around

### ISSUE THREAD

**Issue by gomw** [CitizensDev/Citizens#335]
when i make a npc with my name it starts to follow me around and push me into stuff.. any way you guys can help??

**Comment by fullwall** [comment]
Can you tell me your CB and Citizens version?

**Comment by Viperdream** [comment]
I have the same problem, and when I create an NPC, my commands won't work anymore

I'm using the latest version of bukkit and citizens

**Comment by scandragon** [comment]
Latest is not really a version.
What does /citizens and /version say?

**Comment by dogdark123** [comment]
Same here!
the isssue started when i downloaded build 1337. Both Citizens 1.1 and 1.2

**Comment by Viperdream** [comment]
Sorry about the vague description, I am using 1337 bukkit version and 1.2 for citizens

I now had to remove citizens as well because it caused massive lag. My server has 64 slots and a lot of spare RAM so I don't really know what was going on. Only 3 NPC's were used.

I believe the problem was only with my own NPC, because when I tried to login the "downloading world" part took insanely long and sometimes even timed out, the server crashed because of it as well. My NPC has the same name as me, could this be the issue?
